imageroot/systemd/user/watcher.service

```
@@ -5,6 +5,7 @@ After=network.target
 [Service]
 Type=oneshot
 ExecStart=/usr/bin/systemctl --user reload nethcti-server.service
+ExecStart=/usr/bin/systemctl --user reload reports-api.service
```

The `report-api` service doesn't have the reload, but only restart.
```suggestion
ExecStart=/usr/bin/systemctl --user restart reports-api.service
```